src/crimson/os/seastore/cached_extent.h

```diff
@@ -531,6 +571,37 @@ class CachedExtent
     return ptr.has_value();
   }
 
+  /// Return true if range (offset -> offset + length) is loaded or is about to
```

Comment:
Let's elaborate the comment into 3 cases:
* range fully loaded
* range paritally loaded
* range entirely absent